admin/class-admin-init.php

```diff
@@ -105,8 +105,7 @@ public function tagline_notice() {
 				return;
 			}
 
-			$current_url = ( is_ssl() ? 'https://' : 'http://' );
-			$current_url .= sanitize_text_field( $_SERVER['SERVER_NAME'] ) . sanitize_text_field( $_SERVER['REQUEST_URI'] );
+			$current_url = admin_url( sanitize_text_field( $_SERVER['REQUEST_URI'] ) );
```

Comment:
I would choose for using `filter_input`.
`$current_url = admin_url( sanitize_text_field( filter_input( INPUT_SERVER, 'REQUEST_URI] ) );`